news.js

```diff
@@ -0,0 +1,165 @@
+// Code to display news from News API site. Displays latest headlines by default with ability to search by category and user-specified keyword. Responsive for mobile, tablet and laptop views.
+
+// Variable initialisation
+const myApiKey = "167aa74e22a045b58d8d8af7cb8effe8";
+const myCountry = "gb";
+let myCategory = "";
+let url = `https://newsapi.org/v2/top-headlines?country=${myCountry}&apiKey=${myApiKey}${myCategory}`;
+
+//Utility functions
+function query(eleID) {
+  return document.querySelector(`#${eleID}`);
+}
+function createNode(eleID) {
+  return document.createElement(eleID);
+}
+function append(parent, eleID) {
+  return parent.appendChild(eleID);
+}
+
+// Main function to clear previous content and get new data
+function start(url) {
```

Comment:
function name start is a little opaque. Something more descriptive would work better